distribution/lib/Standard/Table/0.0.0-dev/src/Data_Formatter.enso

```diff
@@ -247,6 +247,9 @@ type Data_Formatter
 
     ## PRIVATE
     get_specific_type_parsers self =
+        ## Have to do date parsing first to allow for pure numeric formats.
+        date_parsers = [self.make_date_time_parser, self.make_date_parser, self.make_time_of_day_parser]
```

Comment:
The comment is appreciated, it would not be at all apparent why/if the ordering is significant. Thanks.